.travis.yml

```diff
@@ -15,7 +15,7 @@ go:
 
 before_install:
   - |
-      if ! git diff --name-only $TRAVIS_COMMIT_RANGE | grep -qvE '(\.md)|(\.png)|(\.pdf)|(\.html)|^(LICENSE)|^(docs)'
+      if ( [ "$TRAVIS_REPO_SLUG" = "kubeedge/kubeedge" ] && ! git diff --name-only $TRAVIS_COMMIT_RANGE ) | grep -qvE '(\.md)|(\.png)|(\.pdf)|(\.html)|^(LICENSE)|^(docs)'
```

Comment:
this change doesn't make sense to me, 
```bash
( [ "$TRAVIS_REPO_SLUG" = "kubeedge/kubeedge" ] && ! git diff --name-only $TRAVIS_COMMIT_RANGE )
```
return a bool, then you grep the bool?